fix: bump music-meta version to correct buf.readInt16BE error; allow spaces in file name;

dh-sesame opened the pull request:
The scanner was failing for a game due to two issues. 
1) `TypeError: buf.readInt16BE is not a function` which seems to be tied to the music-metadata package ( https://github.com/Borewit/music-metadata/issues/856 ) so I set music-metadata to v7.9.1 though possibly would be better to go with latest. I was uncertain of breaking changes down the road or node requirements beyond 10.24.1.
2) A particular game I was running the scanner on is failing due to having asset names that contain spaces. Using "shell-quote" on `filePath` is the proposed solution, but you all may have a preference for a different package. 

I did test these changes using bash on macOS, but am submitting the above more as place markers as I know this is a critical portion of the build pipeline.

902seanryan commented:
I'm comfortable with the `shell-quote` solution, seems pretty simple. 
As well I think leaving the version at 7.9.1 for now is ideal and we can look at updating that in the future when we have some more testing time.

Still have to test this locally myself, but otherwise it looks good to me. 

@deycorinne sound reasonable?

902seanryan reviewed (approved):
Tested locally and it works for me!

dh-sesame commented:
I just added package-lock.json as I noticed the build failed on Github -- had forgotten to add that in when I switched directories locally to create the PR. Just wanted to mention. Hopefully that doesn't bork things. Thanks!
